src/game/Handlers/CharacterHandler.cpp

```diff
@@ -334,10 +334,6 @@ void WorldSession::HandleCharCreateOpcode(WorldPacket& recv_data)
 
         data << (uint8)CHAR_CREATE_SUCCESS;
         SendPacket(&data);
-
-        std::string IP_str = GetRemoteAddress();
-        BASIC_LOG("Account: %d (IP: %s) Create Character:[%s] (guid: %u)", GetAccountId(), IP_str.c_str(), name.c_str(), guidLow);
-        sLog.out(LOG_CHAR, "Account: %d (IP: %s) Create Character:[%s] (guid: %u)", GetAccountId(), IP_str.c_str(), name.c_str(), guidLow);
```

Comment:
Why removing this>

Reply:
It's redundant with the log from Player::SaveNewPlayer.